### ISSUE TITLE
Rename Query to Client

### ISSUE THREAD

**Issue by wingyplus** [wingyplus/dagger-dotnet-sdk#8]

**Comment by pjmagee** [comment]
Yeah this makes sense, I was wondering how this was going to feel / look like when I made the `Connect` function it returned query but i just left it however now you mention this, it makes more sense to me.
